shared/yaml/qlpack.yml

```
@@ -0,0 +1,6 @@
+name: codeql/yaml
+version: 0.0.1-dev
+groups: shared
+library: true
+dependencies: 
+  codeql/util: ${workspace}
```

```suggestion
library: true
```

I don't see any imports of `codeql.util`?